### ISSUE TITLE
Manually Remove Package

### ISSUE THREAD

**Issue by tmcconechy** [rlidwka/sinopia#344]
Hi,

I have a stubborn old package on sinopia v1.4.0 that wont remove itself with npm unpublish. (including with force).

Is there a manual way to go in to the server and get it out of there?

Thanks

**Comment by bjouhier** [comment]
I think I hit the same problem. Same sinopia version: 1.4.0

I published an update of one of my public packages to our local sinopia instead of publishing it to npmjs.org. To fix this mistake, I unpublished --force on sinopia and I republished on npmjs.org (with the --registry option). 

But after that sinopia's proxy was broken: it refused to give me the version that I had republished on npmjs.org.

We deleted the package folder manually on the sinopia server and the problem went away.
